sos/report/plugins/seagate_ses.py

```diff
@@ -0,0 +1,59 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class SeagateSES(Plugin, IndependentPlugin):
```

Comment:
With the new `sos help` command, it's a good idea to add a descriptive docstring to plugin classes as that will get used in the output of `sos help report.plugins.$plugin`.

See the [kernel plugin](https://github.com/sosreport/sos/blob/main/sos/report/plugins/kernel.py) for a good example. You don't need to specify every collection, but give an overview with it.